clonelists/Sony - PlayStation 2.json

```diff
@@ -278,6 +288,28 @@
 		"Bust-A-Bloc": [
 			["Simple 2000 Series Vol. 5 - The Block Kuzushi Hyper", 1]
 		],
+	    	"Buzz! Brain of the World": [
+			["Buzz! Brain of the UK	", 1],
+			["Buzz! Le Plus Malin des Français", 1],
+			["Buzz! Deutschlands Superquiz", 1],
+			["Buzz! Brain of New Zealand", 1],
+			["Buzz! Brain of Oz", 1],
+			["Buzz! Il Quizzone Nazionale", 1],
+			["Buzz! Que sabes de tu Pais", 1],
+			["Buzz! Brain of Switzerland", 1],
+			["Buzz! Le Plus Malin des Belges", 1],
+			["Buzz! De strafste van België", 1],
```

Comment:
Clone lists use the names present in the Redump dat, not the original title. This means things change compared to the original title, like special characters. For instance, ë, ß, ä, é, ê, Сокровища нации, Ł, ł, and ó aren't valid here, the romanized versions are used instead. Colons and other punctuation that aren't valid in file names are also special characters. You'll need to refer to the dat to get the proper names.

Clone lists also represent the current state of the dat, they don't contain every clone that was released. This is because names can change based on Redump's naming rules (sometimes things are romanized, sometimes they're translated... it can be a bit of a trap trying to predict the future here).